<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>408</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Bus seller</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../build-IPZ_LABS-Desktop_Qt_5_4_2_MSVC2012_OpenGL_32bit-Debug/debug/icon.png</normaloff>../build-IPZ_LABS-Desktop_Qt_5_4_2_MSVC2012_OpenGL_32bit-Debug/debug/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#MainWindow{
background-color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 rgba(0, 169, 255, 147), stop:0.497326 rgba(0, 0, 0, 147), stop:1 rgba(0, 169, 255, 147));
}

QGroupBox{
color: rgb(255, 255, 255);
}

QCheckBox{
color: rgb(255, 255, 255);
}

QLabel#label_2{
color: rgb(255, 255, 255);
}

QPushButton#selectButton{
	background-color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:1, fx:0.5, fy:0.5, stop:0 rgba(176, 164, 64, 255), stop:1 rgba(255, 255, 255, 255));
	border-width: 1.5px;
	border-style: outset;
    border-radius: 10px;
 	
	border-color: rgb(255, 255, 187);
	padding: 2px;
	font: 75 10pt &quot;MV Boli&quot;;
	color:rgb(0, 0, 0);
}
QPushButton#selectButton:pressed {
    
	background-color:qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:1.0, fx:0.5, fy:0.5, stop:0 rgba(64, 176, 77, 255), stop:1 rgba(255, 255, 255, 255));
border-style: inset
}

QPushButton#selectButton:hover {
    
	background-color:qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:1.0, fx:0.5, fy:0.5, stop:0 rgba(64, 176, 77, 255), stop:1 rgba(255, 255, 255, 255));
border-style: inset
}

QPushButton#selectButton:disabled {
    
	background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1, fx:0.5, fy:0.46, stop:0 rgba(37, 20, 21, 255), stop:1 rgba(255, 255, 255, 255));
	color: rgb(255, 255, 255);

}

QPushButton#reconnectButton{
	background-color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:1, fx:0.5, fy:0.5, stop:0 rgba(176, 164, 64, 255), stop:1 rgba(255, 255, 255, 255));
	border-width: 1.5px;
	border-style: outset;
    border-radius: 10px;
 	
	border-color: rgb(255, 255, 187);
	padding: 2px;
	font: 75 10pt &quot;MV Boli&quot;;
	color:rgb(0, 0, 0);
}
QPushButton#reconnectButton:pressed {
    
	background-color:qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:1.0, fx:0.5, fy:0.5, stop:0 rgba(64, 176, 77, 255), stop:1 rgba(255, 255, 255, 255));
border-style: inset
}

QPushButton#reconnectButton:hover {
    
	background-color:qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:1.0, fx:0.5, fy:0.5, stop:0 rgba(64, 176, 77, 255), stop:1 rgba(255, 255, 255, 255));
border-style: inset
}

QPushButton#reconnectButton:disabled {
    
	background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1, fx:0.5, fy:0.46, stop:0 rgba(37, 20, 21, 255), stop:1 rgba(255, 255, 255, 255));
	color: rgb(255, 255, 255);

}


QPushButton#nextButton{
	background-color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:1, fx:0.5, fy:0.5, stop:0 rgba(176, 164, 64, 255), stop:1 rgba(255, 255, 255, 255));
	border-width: 1.5px;
	border-style: outset;
    border-radius: 10px;
 	
	border-color: rgb(255, 255, 187);
	padding: 2px;
	font: 75 10pt &quot;MV Boli&quot;;
	color:rgb(0, 0, 0);
}
QPushButton#nextButton:pressed {
    
	background-color:qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:1.0, fx:0.5, fy:0.5, stop:0 rgba(64, 176, 77, 255), stop:1 rgba(255, 255, 255, 255));
border-style: inset
}

QPushButton#nextButton:hover {
    
	background-color:qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:1.0, fx:0.5, fy:0.5, stop:0 rgba(64, 176, 77, 255), stop:1 rgba(255, 255, 255, 255));
border-style: inset
}

QPushButton#nextButton:disabled {
    
	background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1, fx:0.5, fy:0.46, stop:0 rgba(37, 20, 21, 255), stop:1 rgba(255, 255, 255, 255));
	color: rgb(255, 255, 255);

}
QPushButton#changedecisionButtom{
	background-color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:1, fx:0.5, fy:0.5, stop:0 rgba(176, 164, 64, 255), stop:1 rgba(255, 255, 255, 255));
	border-width: 1.5px;
	border-style: outset;
    border-radius: 10px;
 	
	border-color: rgb(255, 255, 187);
	padding: 2px;
	font: 75 10pt &quot;MV Boli&quot;;
	color:rgb(0, 0, 0);
}
QPushButton#changedecisionButtom:pressed {
    
	background-color:qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:1.0, fx:0.5, fy:0.5, stop:0 rgba(64, 176, 77, 255), stop:1 rgba(255, 255, 255, 255));
border-style: inset
}

QPushButton#changedecisionButtom:hover {
    
	background-color:qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:1.0, fx:0.5, fy:0.5, stop:0 rgba(64, 176, 77, 255), stop:1 rgba(255, 255, 255, 255));
border-style: inset
}

QPushButton#changedecisionButtom:disabled {
    
	background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1, fx:0.5, fy:0.46, stop:0 rgba(37, 20, 21, 255), stop:1 rgba(255, 255, 255, 255));
	color: rgb(255, 255, 255);

}

QPushButton#exitButton{
	background-color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:1, fx:0.5, fy:0.5, stop:0 rgba(176, 164, 64, 255), stop:1 rgba(255, 255, 255, 255));
	border-width: 1.5px;
	border-style: outset;
    border-radius: 10px;
 	
	border-color: rgb(255, 255, 187);
	padding: 2px;
	font: 75 10pt &quot;MV Boli&quot;;
	color:rgb(0, 0, 0);
}
QPushButton#exitButton:pressed {
    
	background-color:qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:1.0, fx:0.5, fy:0.5, stop:0 rgba(64, 176, 77, 255), stop:1 rgba(255, 255, 255, 255));
border-style: inset
}

QPushButton#exitButton:hover {
    
	background-color:qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:1.0, fx:0.5, fy:0.5, stop:0 rgba(64, 176, 77, 255), stop:1 rgba(255, 255, 255, 255));
border-style: inset
}

QPushButton#exitButton:disabled {
    
	background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1, fx:0.5, fy:0.46, stop:0 rgba(37, 20, 21, 255), stop:1 rgba(255, 255, 255, 255));
	color: rgb(255, 255, 255);
}

QLCDNumber{
    background-color: rgb(255, 255, 255);
	border-color: rgb(0, 0, 0);
	color: rgb(0, 0, 0);
border-style: inset;
border-width: 2px;
border-radius: 10px;
}

QTableWidget#tableWidget{
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1, fx:0.5, fy:0.5, stop:0 rgba(141, 234, 164, 255), stop:1 rgba(255, 255, 255, 255));
	border-color: rgb(0, 0, 0);
	border-style: outset; 
	border-width: 2px;
	border-radius: 10px;
	color: rgb(0, 0, 0);
	gridline-color: rgb(0, 0, 0);
	selection-background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1, fx:0.5, fy:0.5, stop:0 rgba(141, 144, 87, 255), stop:1 rgba(255, 255, 255, 255));
	selection-color: rgb(255, 255, 255);
}
QTableWidget#tableWidget:disabled {
    
	background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1, fx:0.5, fy:0.46, stop:0 rgba(37, 20, 21, 255), stop:1 rgba(255, 255, 255, 255));
	color: rgb(255, 255, 255);

}
QTableWidget QTableCornerButton::section 
{
border-style: outset; 
	border-radius: 5px;
border-width: 1px;
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1, fx:0.5, fy:0.5, stop:0 rgba(52, 102, 85, 255), stop:1 rgba(255, 255, 255, 255));

gridline-color: rgb(0,0, 0);
}

QHeaderView::section{
border-style: outset; 
	border-radius: 5px;
border-width: 1px;
padding-left: 4px;
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1, fx:0.5, fy:0.5, stop:0 rgba(52, 102, 85, 255), stop:1 rgba(255, 255, 255, 255));
	gridline-color: rgb(0,0, 0);
}
QHeaderView{
padding: 1px;
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1, fx:0.5, fy:0.5, stop:0 rgba(52, 102, 85, 255), stop:1 rgba(255, 255, 255, 255));
gridline-color: rgb(0,0, 0);
		border-color: rgb(0, 0, 0);
}

QComboBox#placeList{
border-style: inset;
border-width: 2px;
border-radius: 0;
border-color: rgb(230, 226, 255);
background-color: qlineargradient(spread:pad, x1:0.387, y1:1, x2:1, y2:0, stop:0 rgba(183, 183, 183, 255), stop:1 rgba(255, 255, 255, 255));
	selection-color: rgb(11, 11, 11);
selection-background-color: qlineargradient(spread:pad, x1:0.387, y1:1, x2:1, y2:0, stop:0 rgba(183, 183, 183, 255), stop:1 rgba(255, 255, 255, 255));
}

QLabel#label{
	color: rgb(255, 255, 255);
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>296</width>
      <height>183</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>431</width>
      <height>248</height>
     </rect>
    </property>
    <property name="title">
     <string>Take your bus and direction</string>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QTableWidget" name="tableWidget">
       <property name="cursor" stdset="0">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="placeList">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="selectButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="nextButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="changedecisionButtom">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Change decision</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>370</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Today is:</string>
    </property>
    <widget class="QLCDNumber" name="dateLCD">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>22</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="clockLcd">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>22</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="logBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>301</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status Logger</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reconnectButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>280</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Reconnect</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ipBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>360</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Use this ip</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Server IP</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>exitButton</tabstop>
  <tabstop>calendarWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
